app/components/header.tsx

```diff
@@ -1,11 +1,66 @@
-import { Text, Center } from '@chakra-ui/react';
+import { Text, Box, Heading, Avatar, Button } from '@chakra-ui/react';
+import { Link, Form } from '@remix-run/react';
+
+import { TriangleUpIcon, LockIcon } from '@chakra-ui/icons';
+import { useUser } from '~/utils';
 
 export default function Header() {
+  const user = useUser();
   return (
     <header>
-      <Center>
-        <Text fontSize="2xl">Hi! I am the header</Text>
-      </Center>
+      <Box
+        display="flex"
+        backgroundColor="brand.500"
+        height="70px"
+        alignItems="center"
+        shadow="2xl"
+        border="2px"
+        fontSize="18"
+      >
+        <Box
+          width="50%"
+          display="flex"
+          color="white"
+          justifyContent="flex-start"
+          gap="10"
+          marginLeft="10"
+        >
+          <Link to={{ pathname: '/domains' }}>
+            <Box display="flex" alignItems="center">
+              <TriangleUpIcon marginRight="2" />
+              <Text>Custom Domains</Text>
+            </Box>
+          </Link>
+
+          <Link to={{ pathname: '/certificate' }}>
+            <Box display="flex" alignItems="center">
+              <LockIcon marginRight="2" />
+              <Text>Certificate</Text>
+            </Box>
+          </Link>
+        </Box>
+        <Link to={{ pathname: '/' }}>
+          <Heading as="h1" size="2xl" color="white">
+            My.Custom.Domains
+          </Heading>
+        </Link>
+        <Box
+          width="50%"
+          display="flex"
+          justifyContent="flex-end"
+          alignItems="center"
+          color="white"
+          gap="5"
+          marginRight="10"
+        >
+          <Text>{user.username}</Text>
+          <Form action="/logout" method="post">
+            <Button _hover={{ bg: '0' }} size="2xs" type="submit">
+              <Avatar size="md" name={user.firstName}></Avatar>
```

Comment:
Because we'll never have an icon, let's use a generic icon instead of the initials, something like these https://chakra-ui.com/docs/components/avatar#customize-the-fallback-avatar

Reply:
![image](https://user-images.githubusercontent.com/32577022/218106552-e07bb12f-1368-4659-b5ae-c6bf13dd4b61.png)

I made changes accordingly let me know what you think